.aws/task-definition-template.json

```diff
@@ -52,6 +52,10 @@
           "name": "LTFT_ASSIGNMENT_UPDATE_TOPIC",
           "valueFrom": "/tis/trainee/notifications/${environment}/topic-arn/ltft/assignment-update"
         },
+        {
+          "name": "FORMR_UPDATE_TOPIC",
+          "valueFrom": "/tis/trainee/forms/${environment}/topic-arn/formr-update"
```

Comment:
The downstream ndw-exporter does need to know which form type it is (part A or part B) since it uses this to determine the NDW datalake filepath. Since we are only using a single SNS for both types, a new message attribute has been included with the form type.